specification/netapp/resource-manager/Microsoft.NetApp/stable/2020-06-01/netapp.json

```diff
@@ -160,6 +160,53 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.NetApp/locations/{location}/checkQuotaAvailability": {
```

Comment:
there is a standard pattern to expose quota details - did you investigate if that works for you?
https://armwiki.azurewebsites.net/api_contracts/UsagesAPIContract.html

Reply:
We cannot change this ATM for the service is live. Ticket created to address this in backend service.